serverless-functions/.env.dev

```diff
@@ -1,9 +1,9 @@
 # ACCOUNT SPECIFIC SETTINGS
 
 # GENERAL 
-TWILIO_FLEX_WORKSPACE_SID=WSdda966af760d30cca8dbd064987673cf
```

Comment:
revert changes to this file

Reply:
I intially removed the SIDs but believe you will be keeping those in as examples